extensions/wikia/ArticleComments/js/ArticleComments.js

```diff
@@ -123,7 +123,8 @@
 					id: commentId,
 					method: 'axEdit',
 					rs: 'ArticleCommentsAjax',
-					useskin: window.skin
+					useskin: window.skin,
+					cb: Date.now()
```

Comment:
Ah, good to know @macbre. Yea, that does return different headers. I don't see the `X-Pass-Cache-Control` header, these are the results, but they are different.

```HTTP
HTTP/1.1 200 OK
Date: Thu, 30 Jul 2015 18:25:51 GMT
Server: Apache
X-Content-Type-Options: nosniff
Status: 200 OK
Last-Modified: Thu, 30 Jul 2015 18:25:51 GMT
Expires: Mon, 26 Jul 1997 05:00:00 GMT
Cache-Control: no-cache, must-revalidate
Pragma: no-cache
Content-Encoding: gzip
Vary: Accept-Encoding
Content-Length: 808
Keep-Alive: timeout=15, max=10000
Connection: Keep-Alive
Content-Type: application/json; charset=utf-8
```

That was after Curling the url and adding the flag `-x ap-s38:80`.

That being said, I'm starting to worry that this PR is starting to languish as it's been open for a week already. Fixing the underlying issue would be preferred, but it's also a relatively edge-casey bug (as I understand it) and we have plenty of more business-critical tickets to work on. To ration our time effectively, I think this cb as a work around is sufficient for now. We can add a TODO as a reminder to address this later if we'd like.